cmd/fetch.go

```diff
@@ -261,6 +275,30 @@ func fetchMesos(cmd *cobra.Command, args []string) {
 	fmt.Println(url)
 }
 
+func fetchMesosLeaderFromAgent(url string) (mesosLeaderHostName string) {
```

Comment:
1. I would like to keep the default zk=localhost. You can check if localhost does not have zk, then  mesos-agent or vice versa.
2. Can we return the error too? I know it may not be user-friendly if we print out the whole error but, we should print out the reason why it cannot reach local agent. Or at least in debug mode.

Reply:
1. Okay, will make it fall back to ZK localhost if local Mesos agent doesn't give the leader.
2. I will log error in debug mode.